src/components/IOUConfirmationList.js

```diff
@@ -328,84 +326,161 @@ class IOUConfirmationList extends Component {
-                        placeholderTextColor={themeColors.placeholderText}
-                    />
-                </View>
-                <FixedFooter>
-                    {this.props.network.isOffline && (
-                        <Text style={[styles.formError, styles.pb2]}>
-                            {this.props.translate('session.offlineMessage')}
-                        </Text>
-                    )}
-                    {shouldShowSettlementButton ? (
-                        <SettlementButton
-                            isDisabled={shouldDisableButton}
-                            isLoading={this.props.iou.loading && !this.props.network.isOffline}
-                            onPress={this.onPress}
-                            shouldShowPaypal={Boolean(recipient.payPalMeAddress)}
-                            recipientPhoneNumber={recipient.phoneNumber}
-                            enablePaymentsRoute={ROUTES.IOU_SEND_ENABLE_PAYMENTS}
-                            addBankAccountRoute={ROUTES.IOU_SEND_ADD_BANK_ACCOUNT}
-                            addDebitCardRoute={ROUTES.IOU_SEND_ADD_DEBIT_CARD}
-                            currency={this.props.iou.selectedCurrencyCode}
-                        />
-                    ) : (
-                        <ButtonWithMenu
-                            isDisabled={shouldDisableButton}
-                            isLoading={isLoading}
-                            onPress={(_event, value) => this.onPress(value)}
-                            options={this.splitOrRequestOptions}
-                        />
-                    )}
-                </FixedFooter>
-            </>
+            <ArrowKeyFocusManager
+                initialFocusedIndex={this.allOptions.length}
+                listLength={this.allOptions.length + 1}
+                onFocusedIndexChanged={this.scrollToIndex}
+                onEnterKeyPressed={this.maybeToggleParticipant}
+                shouldEnterKeyEventBubble={focusedIndex => !this.props.hasMultipleParticipants || !this.allOptions[focusedIndex]}
+            >
+                {({focusedIndex}) => (
+                    <>
+                        <ScrollView style={[styles.flexGrow0, styles.flexShrink1, styles.flexBasisAuto, styles.w100]}>
+                            <OptionsList
+                                ref={e => this.list = e}
+                                sections={this.state.sections}
+                                focusedIndex={focusedIndex}
+                                hideAdditionalOptionStates
+                                forceTextUnreadStyle
+                                canSelectMultipleOptions={this.props.hasMultipleParticipants}
+                                selectedOptions={this.getSelectedOptions()}
+                                onSelectRow={toggleOption}
+                                disableRowInteractivity={!this.props.isGroupSplit}
```

Comment:
Sorry, I don't know what you mean.

Reply:
Basically when making a money request, the participant shouldn't be clickable because it can't be toggled. I hope I'm making sense.

Steps
1. Request money to a user
2. Go next
3. Click on the participant in the IOU confirmation page.

Expected result
The participant is **not** clickable

Actual result
The participant is clickable.

https://user-images.githubusercontent.com/29673073/156633504-4a0e49d3-5432-499e-b4eb-c15cedb6c2c5.mov

Reply:
This is fixed